py3-jupyter-console.yaml

```
@@ -0,0 +1,120 @@
+package:
+  name: py3-jupyter-console
+  version: 6.6.3
+  epoch: 1
```

```suggestion
  epoch: 0
```

This is a new package right?

---

Oh yes, good catch. Thanks

---

Oh yes, good catch. Thanks